<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>encriptador</class>
 <widget class="QDialog" name="encriptador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Criptografar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="n">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continuar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nimbus Sans [UKWN]&quot;;
background-color: rgb(47, 47, 47);
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string> CONTINUAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="e">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="titulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Nimbus Sans [UKWN]&quot;;
border: 2px solid white;</string>
   </property>
   <property name="text">
    <string>DIGITE O TEXTO PARA SER CRIPTOGRAFADO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nimbus Sans [UKWN]&quot;;
border: 1px solid white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
